frontend/src/components/SubscribeMail/SubscribeMail.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import mailImg from '../../assets/images/default-mail.svg'
+import { FaExclamationTriangle } from 'react-icons/fa';
+import './SubscribeMail.styles.sass'
+
+export default function () {
+  const [name, setName] = useState('')
+  const [email, setEmail] = useState('')
+  const [isValid, setIsValid] = useState(true);
+  const [successMessage, setSuccessMessage] = useState('');
+  
+ 
+  const handleSubmit = e => {
+    e.preventDefault();
+    setSuccessMessage('')
+    setIsValid(true)
+    if (name.length <= 3 || email.length <= 3) {
+      setIsValid(false);
```

Comment:
@tamas-w the valid state is not really necessary, you can just have another function called `isValid` that checks the length of the name, email and the format of the email returning a boolean